tracer/src/Datadog.Trace/AppSec/Transports/Http/HttpTransport.Core.cs

```diff
@@ -23,44 +28,38 @@ internal class HttpTransport : ITransport
 
         public Func<string, string> GetHeader => key => context.Request.Headers[key];
 
-        public Request Request()
+        public IContext GetAdditiveContext()
         {
-            var request = new Request
-            {
-                Method = context.Request.Method,
-                Path = context.Request.Path,
-                Scheme = context.Request.Scheme,
-                Url = context.Request.GetUrl()
-            };
-
-            if (context.Request.Host.HasValue)
-            {
-                request.Host = context.Request.Host.ToString();
-                request.RemoteIp = context.Connection.RemoteIpAddress.ToString();
-                request.Port = context.Connection.RemotePort;
-            }
+            return context.Features.Get<IContext>();
```

Comment:
Don't we name all non static private variables with a '_' here? I bugged when I saw the line, I was like.. "Wait, what?!, but there are no variablesin that method" :p. It's monday morning, don't judge me

Reply:
We don't seem to strictly follow the private member variables start with '_' convention in this code base, but it is quite common, so I've adapted both implementations of HttpTransport in 2403b6fd